### PR TITLE
Selmer

### PR DESCRIPTION
**Pull request by fieker**
WIP

but workable

### CODE REVIEW

**Comment by thofma** [comment]
Do you want to add an example to the tests?

**Comment by fieker** [comment]
Oh yes, and do the qq-case as well, and a non-fac-elem

File: src/NumField/Selmer.jl [hunk 1]
```diff
@@ -142,4 +177,122 @@ function pselmer_group_fac_elem(p::Int, S::Vector{<:NfOrdIdl})
...
+      fl, r = is_power_with_root(k, p)
+      fl || return false, a
+      push!(K, r)
+      push!(V, v)
+    end
+  end
+  return true, FacElem(QQ, K, V, parent = parent(a))
+end
+
+#TODO: do we want that?
```
Comment (thofma) [inline src/NumField/Selmer.jl:279]: I would say no.

**Comment by fieker** [comment]
I'll remove it. It came in since fac elem parents in number fields are
cached, but QQ is not...

**Pull request merged**